scripts/renamer/README.md

```diff
@@ -0,0 +1,49 @@
+# Renamer
+
+This module is used to apply renaming rules to service IDs and document types.
```

Comment:
> So is it a module or a script? 🤔

Reply:
It's a module. I'm going to move it in `scripts/utils/renamer` to avoid confusion.